common/persistence/pinotVisibilityTripleManager.go

```diff
@@ -397,23 +397,32 @@ func (v *pinotVisibilityTripleManager) CountWorkflowExecutions(
 
 func (v *pinotVisibilityTripleManager) chooseVisibilityManagerForRead(domain string) (VisibilityManager, VisibilityManager) {
 	var visibilityMgr VisibilityManager
-	var visibilityMgr2 VisibilityManager
```

Comment:
I think this is just for debugging purpose to read from Pinot, we can change this part after we redesign the comparator